conf/firewalld/firewalld_services.conf.defaults

```diff
@@ -0,0 +1,513 @@
+[haproxy-admin]
+short=packetfence-haproxy-admin
+description=Firewalld ports for pf-admin service
+helper_name=haproxy-admin
+ports=haproxy-admin_port1,haproxy-admin_port2
+
+[haproxy-admin_port1]
+port=1443
+type=tcp
+[haproxy-admin_port2]
+port=1443
+type=udp
+
+[web-portal]
+short=packetfence-httpd.portal
+description=Firewalld ports for httpd.portal service
+ports=web-portal_port1
+
+[web-portal_port1]
+port=8080
+type=tcp
+
+[proxysql]
+short=proxysql
+description=Firewalld ports for proxysql service
+ports=proxysql_port1,proxysql_port2,proxysql_port3,proxysql_port4
+
+[proxysql_port1]
+port=6033
+type=udp
+[proxysql_port2]
+port=6033
+type=tcp
+[proxysql_port3]
+port=6080
+type=udp
+[proxysql_port4]
+port=6080
+type=tcp
+
+[radius_acct]
+short=radius acct port
+description=Firewalld ports for radius acct service
+ports=radius_acct_port1,radius_acct_port2
+
+[radius_acct_port1]
+port=1813
+type=udp
+[radius_acct_port2]
+port=1813
+type=tcp
+
+[radius_acct_clu]
+short=radius acct cluster port
+description=Firewalld ports for radius acct service
+ports=radius_acct_clu_port1,radius_acct_clu_port2
+
+[radius_acct_clu_port1]
+port=1823
+type=udp
+[radius_acct_clu_port2]
+port=1823
+type=tcp
+
+[radius_auth]
+short=radius auth port
+description=Firewalld ports for radius auth service
+ports=radius_auth_port1,radius_auth_port2,radius_auth_port3
+
+[radius_auth_port1]
+port=1812
+type=udp
+[radius_auth_port2]
+port=1812
+type=tcp
+[radius_auth_port3]
+port=2083
+type=tcp
+
+[radius_auth_clu]
+short=radius auth cluster port
+description=Firewalld ports for radius auth service
+ports=radius_auth_clu_port1,radius_auth_clu_port2,radius_auth_clu_port2
+
+[radius_auth_clu_port1]
+port=1822
+type=udp
+[radius_auth_clu_port2]
+port=1822
+type=tcp
+[radius_auth_clu_port3]
+port=2093
+type=tcp
+
+[radius_cli]
+short=radius cli port
+description=Firewalld ports for radius cli service
+ports=radius_cli_port1,radius_cli_port2
+
+[radius_cli_port1]
+port=1815
+type=udp
+[radius_cli_port2]
+port=1815
+type=tcp
+
+[radius_cli_clu]
+short=radius cli cluster port
+description=Firewalld ports for radius cli service
+ports=radius_cli_clu_port1,radius_cli_clu_port2
+
+[radius_cli_clu_port1]
+port=1825
+type=udp
+[radius_cli_clu_port2]
+port=1825
+type=tcp
+
+[radius_eduroam]
+short=radius eduroam port
+description=Firewalld ports for radius eduroam service
+ports=radius_eduroam_port1,radius_eduroam_port2
+
+[radius_eduroam_port1]
+port=11812
+type=udp
+[radius_eduroam_port2]
+port=11812
+type=tcp
+
+[radius_eduroam_cl]
+short=radius eduroam cluster port
+description=Firewalld ports for radius eduroam service
+ports=radius_eduroam_cl_port1,radius_eduroam_cl_port2
+
+[radius_eduroam_cl_port1]
+port=11822
```

Comment:
11812

Reply:
btw this value is coming from the config

Reply:
I took care of that too in the function.
See that as a default.

Reply:
This is also the cluster part.

Reply:
https://github.com/inverse-inc/packetfence/blob/feature/INV-41/lib/pf/firewalld.pm#L811